src/General/Alert/Alert.tsx

```diff
@@ -18,11 +18,11 @@ import {
 
 import { PrimaryColor, SecondaryColor } from '../../Utils/Colors';
 
-const Alert = ({
+export const Alert = ({
   isOpen,
-  autoClose,
+  autoClose = 3000,
```

Comment:
Oh why we need to set the default for `autoClose`?

Reply:
Hi @rosslo, oh since `autoClose` it's not a mandatory field, if default `autoClose` is not specified then the alert will be open indefinitely (unless user wants to close it), thus is that the wanted behaviour?

Reply:
Hmm, I think there might be a case is that we want to ensure the users see the message. So we don't auto close alert until the users see the message and close it by themseleves.

Reply:
Make sense got it 🙏